README.md

```diff
@@ -2,25 +2,13 @@
 
 ## Include the SDK for iOS in an Existing Application
 
-There are three ways to import the DroneCore-Swift SDK for iOS into your project:
-
-* [CocoaPods](https://cocoapods.org/)
-* Manually
-
-You should use one of these three ways to import the DroneCore-Swift SDK but not multiple. Importing the SDK in multiple ways loads duplicate copies of the SDK into the project and causes compiler errors.
-
 ### CocoaPods
 
-1. The DroneCore-Swift SDK for iOS is available through [CocoaPods](http://cocoapods.org). If you have not installed CocoaPods, install CocoaPods by running the command:
+1. The DroneCore-Swift SDK for iOS is available through [CocoaPods](http://cocoapods.org). CocoaPods can be installed with the following commands:
```

Comment:
Should this also be renamed to DronecodeSDK-Swift?

Reply:
Yes, I'll need to change the podspec and push it to the Cocoapods repo. I'll do that later (ASAP).